### PR TITLE
gdk: Fixes for keymap/keysym API

### PR DESCRIPTION
**Pull request by ids1024**

### CODE REVIEW

File: gdk/src/keys.rs [hunk 1]
```diff
@@ -79,6 +92,19 @@ impl Key {
         skip_assert_initialized!();
         unsafe { ffi::gdk_keyval_to_lower(**self) }.into()
     }
+
+    #[doc(alias = "gdk_keyval_convert_case")]
+    pub fn keyval_convert_case(&self) -> (Self, Self) {
```
Comment (bilelmoussaoui) [inline gdk/src/keys.rs:97]: Maybe rename this one to just convert_case?

Comment (ids1024) [reply gdk/src/keys.rs]: Ah right, forgot to rename that.

File: gdk/Gir.toml [hunk 2]
```diff
@@ -183,6 +183,34 @@ status = "generate"
...
+    #manual
+    ignore = true
+    [[object.function]]
+    name = "keyval_convert_case"
+    #manual
+    ignore = true
+    [[object.function]]
+    name = "keyval_is_lower"
+    #manual
+    ignore = true
```
Comment (sdroege) [inline gdk/Gir.toml:200]:
```suggestion
    manual = true
```

etc

File: gdk/src/keys.rs [hunk 3]
```diff
@@ -44,6 +45,18 @@ impl IntoGlib for Key {
 }
 
 impl Key {
+    #[doc(alias = "gdk_keyval_from_name")]
+    pub fn from_name(keyval_name: &str) -> Self {
```
Comment (sdroege) [inline gdk/src/keys.rs:49]: This can't fail?

File: gdk/src/keys.rs [hunk 4]
```diff
@@ -44,6 +45,18 @@ impl IntoGlib for Key {
...
+        skip_assert_initialized!();
+        Self(unsafe { ffi::gdk_keyval_from_name(keyval_name.to_glib_none().0) })
+    }
+
+    #[doc(alias = "gdk_unicode_to_keyval")]
+    pub fn from_unicode(c: char) -> Self {
```
Comment (sdroege) [inline gdk/src/keys.rs:55]: This can't fail?

Comment (ids1024) [reply gdk/src/keys.rs:55]: It returns "the corresponding GDK key symbol, if one exists. or, if there is no corresponding symbol, wc | 0x01000000". I'm not really sure how that is supposed to make sense, but I guess it doesn't count as a "failure" and is a valid "keyval"?

File: gdk/src/keys.rs (hunk 3 above)
Comment (ids1024) [reply gdk/src/keys.rs:49]: The documentation says it returns "`GDK_KEY_VoidSymbol` if the key name is not a valid key". So I guess not a failure? This is different from `gdk_keymap_lookup_key` which returns `0` if no mapping exists.

Comment (sdroege) [reply gdk/src/keys.rs:49]: Strange API, but ok :)

**Pull request merged**